### ISSUE TITLE
net-mgmt/zabbix-agent: Allow CIDR Notation in "Zabbix Servers" field

### ISSUE THREAD

**Issue by SamForma** [opnsense/plugins#2296]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[-] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
yup

[-] I have searched the existing issues and I'm convinced that mine is new.
yup

[-] When the request is meant for an existing plugin, I've added its name to the title.
yup

**Is your feature request related to a problem? Please describe.**
the "Zabbix Servers" field in the Zabbix Servers Agent config will not allow CIDR notation. for larger installations its easier to add a subnet than individual hosts.

**Describe the solution you'd like**
Allow CIDR notation for the "Zabbix Server" field in the Zabbix Agent plugin. Currently it only allows an IP address or hostname. I'd like to use CIDR (#.#.#.#/#) but the form validation will not allow.

**Describe alternatives you've considered**
manually editing the Zabbix Agent config file

**Additional context**
OpnSense: 21.1.3_3
Zabbix Agent (os-zabbix-agent): 1.8_1

**Comment by fraenki** [comment]
Interesting, I was not aware that CIDR notation can be used. I will add it in one of the next releases.

For my own convenience, a quote from the official docs:

```
List of comma delimited IP addresses, optionally in CIDR notation, or hostnames
of Zabbix servers and Zabbix proxies.
Incoming connections will be accepted only from the hosts listed here.
'0.0.0.0/0' can be used to allow any IPv4 address.
Example: Server=127.0.0.1,192.168.1.0/24,::1,2001:db8::/32,zabbix.domain
```

**Comment by fraenki** [comment]
This will be available in the upcoming release 1.9 of os-zabbix-agent.
